### ISSUE TITLE
workload-fn with java lambda expression cannot access external maven dependencies

### ISSUE THREAD

**Issue by mikub** [mikub/titanoboa#5]
Additional [external maven dependencies](https://github.com/mikub/titanoboa/wiki/Server-Configuration#external-dependencies) loaded during runtime are  accessible to all [workflow step functions](https://github.com/mikub/titanoboa/wiki/Designing-Workflows#workload-fn) - i.e. either to clojure library functions, java classes or clojure anonymous functions. 

**They however do not seem to be visible to java lambda function code** (see 4 in [wiki](https://github.com/mikub/titanoboa/wiki/Designing-Workflows#workload-fn))

Steps to reproduce:
1. I load new maven dependencies (using GUI or just by updating ext-dependencies file) during runtime.
2. I then prototype a new workflow step with a java lambda expression and reference the library loaded in step 1
3. I get an error:
```
Compiler messages:
ERROR: package xxx.xxx.xxx does not exist
Compiler standard error output:
```

Expected behaviour: no error in step 3, instead the java lambda code should see the library on the classpath.

Example:
1. load new maven dependency `[com.github.librepdf/openpdf "1.3.11"]`
2. create a new workflow step with following java expression:
```java
p -> { 
        com.lowagie.text.Document doc = new com.lowagie.text.Document();
        java.util.Map m = new java.util.HashMap();
        m.put("doc", doc);
        return m;
      }
```
3. error message is as follows:
```
Compiler messages:
ERROR: package com.lowagie.text does not exist
Compiler standard error output:
```
If I don't use java lambda and just switch to clojure fn I can instantiate `com.lowagie.text.Document` without issues:
```clojure
(fn [p]
{:doc (com.lowagie.text.Document.)})
```

**Comment by mikub** [comment]
Fixed in commit 880ea80d7dcaedd1b2c277ba5e2720e806d17f21

**Comment by mikub** [comment]
This change breaks java lambda on Java 9 and higher though
